src/components/Scheduling/AdvancedScheduling.vue

```diff
@@ -111,6 +123,25 @@ const handleObjectSelection = (object) => {
   emitSelections()
 }
 
+const handleCadencePayload = (payload) => {
+  cadencePayload.value = payload
+  emitSelections()
+}
+
+watch(canAddCadence, (val) => {
+  emits('showGenerateCadence', val)
```

Comment:
I don't like this - we already have an emit for if this is a cadence request when things are filled out so I feel like that should be able to cover it in the parent.